src/HtmlFormatter.js

```diff
@@ -0,0 +1,128 @@
+                           '<td>' + fileStats.lines + ' Lines</td></tr>\n' +
+                           '<tr>\n' +
+                           '<td style="font-size:20px">Errors:</td>\n' +
+                           '<td style="font-size:20px;color:red">' + resultStats.errors + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.errors/fileStats.files).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.errors/fileStats.modules).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.errors/fileStats.lines).padEnd(15, ' ') + '</td>\n' +
+                           '</tr><tr>\n' +
+                           '<td style="font-size:20px">Warnings:</td>\n' +
+                           '<td style="font-size:20px;color:orange">' + resultStats.warnings + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.warnings/fileStats.files).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.warnings/fileStats.modules).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.warnings/fileStats.lines).padEnd(15, ' ') + '</td>\n' +
+                           '</tr><tr>\n' +
+                           '<td>Suggestions:</td>\n' +
+                           '<td>' + resultStats.suggestions + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.suggestions/fileStats.files).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.suggestions/fileStats.modules).padEnd(15, ' ') + '</td>\n' +
+                           '<td>' + fmt.format(resultStats.suggestions/fileStats.lines).padEnd(15, ' ') + '</td>\n' +
+                           '</tr><tr>\n' +
+                           '<td>I18N Score(0-100)</td>\n' +
+                           '<td>' + fmt.format(score) + '</td>\n' +
+                           '</tr>\n' +
+                           '</thead>\n' +
+                           '</table>\n' +
+                           '<br/>\n' +
+                           '<hr align="left" width="1100px"/>\n' +
+                           '<p style="color:#714AFF;text-align:left;font-size:30px;font-weight:bold" width=320px; >Detailed Information</p>\n';
+        return summaryTable;
+    }
+
+    format(result) {
+        let levelTextColor = (result.severity === "error") ? "color:red;" : "color:orange;";
+        let autofix = (result.fix === undefined) ? "unavilable" : result.fix.applied;
+
+        let htmlText = '<table>\n' +
+                       '<thead>\n' +
+                       '<tr><th style=' + levelTextColor + "text-align:left;font-size:22px; width=280px;>" + "["+result.severity + "]"+ "</th><th style='text-align:left'></th></tr>\n" +
+                       '<tr><td style="text-align:left;color:Cadetblue;font-weight:bold">filepath</td><td style="color:Cadetblue;font-weight:bold">' + result.pathName + "</td></tr>\n" +
```

Comment:
Should put the location information in here as well to make it easier to find the problems in the original file.

Reply:
Regarding the linking of the real file path, I'll work on it different PR